folium/features.py

```diff
@@ -49,12 +49,40 @@ def __init__(self, url, name=None,
 
 
 class RegularPolygonMarker(MacroElement):
-    def __init__(self, location, popup=None,
-                 color='black', opacity=1, weight=2,
+    def __init__(self, location, color='black', opacity=1, weight=2,
                  fill_color='blue', fill_opacity=1,
-                 number_of_sides=4, rotation=0, radius=15):
-        """
-        TODO docstring here
+                 number_of_sides=4, rotation=0, radius=15, popup=None):
+        """Custom markers using the Leaflet Data Vis Framework.
+        For more information, see https://humangeo.github.io/leaflet-dvf/
```

Comment:
I was thinking that this should be at the bottom.  That is consistent with the other docstrings I am writing and makes sense.  You read what is here first, then you follow the URL for more info :wink: